verl/trainer/ppo/ray_trainer.py

```diff
@@ -545,7 +545,7 @@ def _create_dataloader(self):
                                          max_prompt_length=self.config.data.max_prompt_length,
                                          filter_prompts=True,
                                          return_raw_chat=self.config.data.get('return_raw_chat', False),
-                                         truncation='error')
+                                         truncation=self.config.data.truncation)
```

Comment:
```suggestion
                                         truncation=self.config.data.truncation
                                         filter_overlong_prompts=self.config.data.filter_overlong_prompts)
```

could you kindly add `filter_overlong_prompts` option  as well?